foundry/foundry.py

```diff
@@ -100,19 +114,35 @@ def load(self, name, download=True, globus=True, verbose=False, **kwargs):
         -------
             self
         """
+        res = []
         # MDF specific logic
-        res = self.forge_client.match_field(
-            "mdf.organizations", "foundry"
-        ).match_resource_types("dataset")
-        res = res.match_field("mdf.source_id", name).search()
+        # Handle DOI inputs
+        if name.startswith("10.") and provider == "MDF":
```

Comment:
will DOIs always start with "10"?

Reply:
Yes, DOIs always start with the string `10. I have no idea why they do that though...